src/opm/parser/eclipse/EclipseState/Grid/FieldPropsManager.cpp

```diff
@@ -141,6 +142,10 @@ bool FieldPropsManager::tran_active(const std::string& keyword) const {
     return this->fp->tran_active(keyword);
 }
 
+void FieldPropsManager::applyNumericalAquifer(const NumericalAquifers& aquifers) {
```

Comment:
Have tried to use `snake_case` consistently in the `FieldProps` class - please rename to e.g. `FieldPropsManager::apply_aquifers()`